utils/_context/containers.py

```diff
@@ -466,3 +474,34 @@ def __init__(self, host_log_folder) -> None:
             host_log_folder=host_log_folder,
             healthcheck={"test": "/healthcheck.sh", "retries": 60},
         )
+
+
+class OpenTelemetryCollectorContainer(TestedContainer):
+    def __init__(self, host_log_folder) -> None:
+        super().__init__(
+            image_name="otel/opentelemetry-collector-contrib:latest",
+            name="collector",
+            command="--config=/etc/otelcol-config.yml",
+            environment={
+                "DD_API_KEY": os.environ.get("DD_API_KEY_3"),
+                "DD_SITE": os.environ.get("DD_SITE", "datad0g.com"),
+            },
+            volumes={"./utils/build/docker/otelcol-config.yaml": {"bind": "/etc/otelcol-config.yml", "mode": "ro",}},
+            host_log_folder=host_log_folder,
+            ports={"13133/tcp": ("0.0.0.0", 13133)},
+        )
+
+    # Override wait_for_health because we cannot do docker exec for container opentelemetry-collector-contrib
+    def wait_for_health(self):
```

Comment:
Can't we do this in `docker-compose` ?

Reply:
`docker-compose` has been removed in https://github.com/DataDog/system-tests/pull/958/files#diff-e45e45baeda1c1e73482975a664062aa56f20c03dd9d64a827aba57775bed0d3